datapipelines/src/main/kotlin/io/customer/datapipelines/extensions/JsonExtensions.kt

```diff
@@ -0,0 +1,48 @@
+package io.customer.datapipelines.extensions
+
+import com.segment.analytics.kotlin.core.utilities.toJsonElement
+import kotlinx.serialization.json.JsonArray
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonNull
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.buildJsonArray
+import kotlinx.serialization.json.buildJsonObject
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * Transforms [JSONObject] to [JsonObject] by leveraging kotlinx.serialization.json.
+ * This is useful when we receive JSONObject for migration tasks and we want to convert
+ * it to JsonObject for processing.
+ */
+fun JSONObject.toJsonObject(): JsonObject = buildJsonObject {
+    val keys = keys()
+    while (keys.hasNext()) {
+        val key = keys.next()
+        val value = get(key)
+        put(key, value.toSerializableJson())
+    }
+}
```

Comment:
```suggestion
fun JSONObject.toJsonObject(): JsonObject = buildJsonObject {
    for (key in keys()) {
        put(key, opt(key).toSerializableJson())
    }
}
```

`get` if you want error to be thrown

Reply:
Updated